AppCenterAuth/AppCenterAuth/MSAuth.m

```diff
@@ -298,16 +299,26 @@ - (void)configAuthenticationClient {
 
   // Init MSAL client application.
   NSError *error;
-  MSALAuthority *auth = [MSALAuthority authorityWithURL:(NSURL * __nonnull) self.authConfig.authorities[0].authorityUrl error:nil];
+  NSURL *authUrl = nil;
+  for (MSALAuthority *auth in self.authConfig.authorities) {
```

Comment:
The `authorities` in `authConfig` is `MSAuthority` type which is defined in App Center.
You can't cast it to `MSALAuthority`, this is dead code and `authUrl` will always be `nil`.
The task is just using `MSALB2CAuthority` instead of `MSALAuthority` so you just need to replace the type from `MSALAuthority` to `MSALB2CAuthority`. @clpolet FYI.